lib/counter.js

```diff
@@ -25,6 +25,17 @@ function removeComments(str) {
     return str;
 }
 
+/**
+ * Empty the string literals. Do lazy match. Avoids removing
+ * everything after '//' in a string literal.
+ */
+function emptyStringLiterals(str) {
+    str = str || "";
+    str = str.replace(/\"[^\n\r]*?\"/g, "''");
```

Comment:
Yep. Some of the regexes in that post are broken. The correct regex for removing quoted strings is

```
/(['"])(?:\\\1|.)+?\1/g
```